jstests/noPassthrough/virtual/error_report_csv_file.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Tests error report over virtual collections over external CSV file(s).
+ */
+
+import {
+    getPlanStage,
+} from "jstests/libs/analyze_plan.js";
+
+removeFile("/tmp/error_report.txt");
+copyFile(pwd() + "/jstests/noPassthrough/virtual/error_report.txt", "/tmp/error_report.txt");
+
+const conn = MongoRunner.runMongod();
+
+const db = conn.getDB(jsTestName());
+db.dropDatabase();
+
+const coll = db.ext_csv;
+
+(function CollectEachErrorCases() {
```

Comment:
[nit] always start with lower case letter for function namaes.

Reply:
Done